sdk/identity/identity/src/tokenCache/InMemoryPersistence.ts

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as msalCommon from "@azure/msal-common";
+import { CachePlugin } from "./types";
+
+/**
+ * To allow silent authentications on the same credential, we provide a very simple in memory token cache.
+ * It can't be used to re-use the account information returned from the authenticate() method.
+ * @internal
+ */
+export class InMemoryPersistence {
+  private inMemoryCache: string = "";
```

Comment:
since this is just TS "private" it's actually a public-to-JS property on the class. Maybe it would be better to make this a module variable instead so it won't be quite so easy to snoop on?

Reply:
should the cache be saved as a string or bytes?

Reply:
This class is not exposed to the public, and it should be removed, since MSAL should be the one doing the work here.

Also, MSAL uses strings, not bytes!

Reply:
I'll move this to exist only within the scope of the `register` method.